### PR TITLE
feat(edge) enable signature checking for edge agent  EE-743

### PR DESCRIPTION
**Pull request by mcpacino**

### CODE REVIEW

File: http/handler/key/handler.go [hunk 1]
```diff
@@ -26,9 +26,9 @@ func NewHandler(notaryService *security.NotaryService, edgeManager *edge.Manager
 	}
 
 	h.Handle("/key",
-		notaryService.DigitalSignatureVerification(httperror.LoggerHandler(h.keyInspect))).Methods(http.MethodGet)
+		httperror.LoggerHandler(h.keyInspect)).Methods(http.MethodGet)
```
Comment (chiptus) [inline http/handler/key/handler.go:29]: why was this removed?

**Review by chiptus — APPROVED** [review]
Looks good. There's one question about the removal of notaryService.

File: http/handler/key/handler.go (hunk 1 above)
Comment (cmenginnz) [reply http/handler/key/handler.go:29]: `/key` endpoint is used by other agents who do not have the signature, so remove the verification for it.
